### ISSUE TITLE
Arabic translation for the theme features

### ISSUE THREAD

**Issue by TheAwiteb** [welpo/tabi#383]
I want to add Arabic translation for the theme, beside English, Català and Español. What do you think?

### Why?
To show how the theme support RTL languages.

### What about the maintaining?
I'll maintain the Arabic version, just mention me and I'll update it soon as possible.

**Comment by welpo** [comment]
Hey Awiteb!

I think it's a fantastic idea. That would really show the possibilities of tabi's i18n.

My only concern is…

> I'll maintain the Arabic version, just mention me and I'll update it soon as possible.

Are you sure about this? I really appreciate the offer, but I don't want to impose too much work. Just translating the entire Mastering tabi post will take quite some time.

Note: I wouldn't expect human-translation on everything. What I generally do is write the docs in English, machine translate (DeepL, Claude, ChatGPT, or Google Translate) to Catalan/Spanish, and make the necessary (often minor) edits.

P.S: Thank you so much for the sponsorship! I really appreciate it~

**Comment by TheAwiteb** [comment]
> Are you sure about this?

Yes, I use the theme on my personal blog, this is the least I can do.

> I wouldn't expect human-translation on everything.

I don't like AI/Google translations, they write very bad Arabic that lacks many things, I will write it myself. I think I will finish it this week.

By the way, I hope you don't translate anything Arabic through them, for example, the "by_author" translation is incorrect (I will correct it). It means "by pen" and writing in the blog is definitely not done with a pen. Just mention me on any Arabic thing and I will done it.

**Comment by welpo** [comment]
> I hope you don't translate anything Arabic through them

When I add a feature I do use the AI translations. I provide context to reduce the probability of wrong translations, but it's not perfect, as you've pointed out.

I'll try to remember to ping you to check future translations :)

> Yes, I use the theme on my personal blog, this is the least I can do.

Your PRs, reviews, starring the project, and the sponsorship are certainly more than enough!

I really appreciate the support. If you insist, I'll gladly take a PR (can be incomplete) with the Arabic translations.
